pages/transactions/index.vue

```diff
@@ -10,7 +10,7 @@
         :data="item.tx"
       />
     </TxList>
-    <LoadMoreButton />
+    <LoadMoreButton @click.native="loadmore"/>
```

Comment:
artem updated this component to `$emit` an event called `@update` then it can also be more semantical:
`@update="transactionList"`. On the level of this page, it should already be apparent what is loaded when reading on the component, and not having to go down to methods and see exactly what the genericly named method does